woocommerce/google-listings-and-ads
# Migrate column data type in MC account Issues table

### Changes proposed in this Pull Request:

Closes #1694.

This PR migrates the data type for `wp_gla_merchant_issues`.`action` 
We migrate from VARCHAR(100) to TEXT

Context: The action column was not used and being defined as 100 Chars limit. Now the issues action contains the description... so sometimes the description seems truncated.

### Screenshots:

Before:
<img width="605" alt="Screenshot 2022-09-23 at 11 41 13" src="https://user-images.githubusercontent.com/5908855/191934007-beea7e1d-65b5-4785-b261-7ea6f3f88046.png">

After:
<img width="748" alt="Screenshot 2022-09-23 at 13 35 12" src="https://user-images.githubusercontent.com/5908855/191934039-d106cd32-0ec6-4a2e-9aa7-70fccef02062.png">


### Detailed test instructions:


1. Change the version in `src/DB/Migration/Migration20220923T1663910364.php:get_applicable_version()` to match the version defined in the plugin file.
2. Enter to the plugin dashboard
3. Check the Table data type in `wp_gla_merchant_issues`.`action`  was changed to TEXT
4. Check that you can add more than 100 characters in the column
5. Check that Product Feed Issues is working as before (if action is set it shows a modal after clicking on Read more about this issue) 


### Changelog entry

> Fix - Avoid truncate for issues with more than 100 characters length
